src/hooks/useKeyboardNavigation.js

```diff
@@ -108,12 +242,22 @@ const useKeyboardNavigation = ({
 
   const focusNextEmoji = () => {
     const nextSibling = closestEmoji().nextElementSibling;
-    if (nextSibling) nextSibling.firstChild.focus();
+    if (nextSibling) {
+      nextSibling.firstChild.focus();
+      return true;
+    }
+
+    return false;
```

Comment:
So, I understand why you return a boolean here. You use it as a "shortcut" for the conditionals above, but that's a bit weird. First the name: `focusNextEmoji`. Nowhere does it hint that it returns a boolean. Same for `focusPrevEmoji`. In general, I am not a big fan of functions that promise you one thing (focus emoji) and return a seemingly unrelated value.
 
A decent tradeoff (while still not perfect) would be to return the actual element instead of returning a boolean. It makes sense that if you focus some element, you also return it.

```suggestion
    if (nextSibling) {
      nextSibling.firstChild.focus();
    }

    return nextSibling;
```

This requires no changes where you actually use these functions.

A more verbose option would be to have two functions, and use them together in each place you use this function:

```
const nextEmoji = getNextEmoji();

if (nextEmoji) {
  focus(nextEmoji);
}
```

This requires some refactors above, and will add quite a lot of code - so we can take it as an improvement for later, but it is definitely the more explicit approach.